examples/machine_vision_object_pickup/main.py

```diff
@@ -0,0 +1,89 @@
+# Automata Technologies - Andrea Antonello, 2019
+import socket
+from automata import Eva
+from evaUtilities import *
+
+# Connection to robot
+host = <user_input>
```

Comment:
Maybe extract all the configuration values that are input from the user into another file for ease of use (having `waypoint_home` in the middle is easy to miss).
A user should also be able to run the example, even with bad settings.

Reply:
It makes sense to have the user parameters in another function.  However, the idea over having the user to manually input the values is to avoid people crashing their robot into existing hardware when using the "bad settings"